tests/components/nexia/test_switch.py

```diff
@@ -0,0 +1,11 @@
+"""The switch tests for the nexia platform."""
+
+from homeassistant.const import STATE_ON
+
+from .util import async_init_integration
+
+
```

Comment:
Note that the tests for this integration currently use requests mocking which should be replace with mocking the library instead.  Refactoring that would be quite large and way beyond the scope of this PR.  I'm planning on doing some more cleanups to the underlying library before I get there.